src/AcquiaDrupalEnvironmentDetector.php

```diff
@@ -222,11 +222,32 @@ public static function getSiteName($site_path) {
     return str_replace('sites/', '', $site_path);
   }
 
+  /**
+   * Is this a Lando environment using the Acquia recipe.
+   */
+  public static function isAcquiaLandoEnv() {
+    return getenv('AH_SITE_ENVIRONMENT') === 'LANDO';
+  }
+
+  /**
+   * Is this a Lando environment.
+   */
+  public static function isLandoEnv() {
+    return getenv('LANDO') === 'ON';
+  }
+
+  /**
+   * Get Lando info.
+   */
+  public static function getLandoInfo() {
+    return getenv('LANDO_INFO');
+  }
+
   /**
    * If this isn't a Cloud environment, assume it's local.
    */
   public static function isLocalEnv() {
-    return !self::isAhEnv();
+    return !self::isAhEnv() || self::isAcquiaLandoEnv();
```

Comment:
I'm fine with this change, but is it intentional? Last I heard it was a problem if `isLocalEnv()`returns true in Lando.

If this works, then awesome.

Reply:
Yes this is intentional. @mikemadison13 confirmed this works.